mantidproject/mantid
# Initial feedback on ILL indirect reductions.

This PR adds few new options and minor fixes reported by the users of ILL indirect reductions.

**To test:**



1. Review and existing tests pass
2. Set ILL/IN16B as facility/instrument
3. Open Interfaces->Indirect->Data Reduction->ILL Energy Transfer
4. Select 136558.nxs from unit tests and click run
5. Play with the Spectrum Axis new options 
6. Check the spectrum axis of the outputs to be in correct units, and have corresponding suffix in the name

Fixes #19220 .

Does not need to be in the release notes.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
